core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java

```diff
@@ -222,31 +244,48 @@ public void setFileNameSessionKey(String fileNameSessionKey) {
 		setFilenameSessionKey(fileNameSessionKey);
 	}
 
-	@IbisDoc({"name of the session key containing the file name of the file containing the result message", ""})
+	/**
+	 * Name of the session key containing the file name of the file containing the result message.
+	 * Path is relative to the configuration's root directory (file is expected on the classpath).
+	 */
 	public void setFilenameSessionKey(String filenameSessionKey) {
 		this.filenameSessionKey = filenameSessionKey;
 	}
 
-	@IbisDoc({"returned message", ""})
+	/**
+	 * Returned message.
+	 */
 	public void setReturnString(String returnString) {
 		this.returnString = returnString;
 	}
 
-	/** When set, any match in the result string will be replaced by the value of <code>replaceTo</code> attribute */
+	/**
+	 * If set, every occurrence of this attribute's value is replaced by the value of <code>replaceTo</code>.
+	 */
 	public void setReplaceFrom(String replaceFrom){
 		this.replaceFrom=replaceFrom;
 	}
 
-	/** Target value for the matches of attribute <code>replaceFrom</code> in the result string */
+	/**
+	 * See <code>replaceFrom</code>.
+	 */
 	public void setReplaceTo(String replaceTo){
 		this.replaceTo=replaceTo;
 	}
 
+	/**
+	 * File name of XSLT stylesheet to apply.
+	 * Path is relative to the configuration's root directory (file is expected on the classpath).
+	 */
```

Comment:
```suggestion
	/**
	 * File name of XSLT stylesheet to apply.
	 */
```
Het pad is inderdaad standaard relatief aan de configuratie root directory. Dat is altijd, overal zo. Ik vind het onhandig dat bij elk attribuut te zetten.

Reply:
Verwerkt.